azure-application-insights-spring-boot-starter/src/main/java/com/microsoft/applicationinsights/autoconfigure/ApplicationInsightsTelemetryAutoConfiguration.java

```diff
@@ -69,8 +73,7 @@
         ApplicationInsightsWebModuleConfiguration.class
 })
 @AutoConfigureBefore(name = {
-    "io.micrometer.spring.autoconfigure.export.azuremonitor.AzureMonitorMetricsExportAutoConfiguration",
-    "org.springframework.boot.actuate.autoconfigure.metrics.export.azuremonitor.AzureMonitorMetricsExportAutoConfiguration"
```

Comment:
I have removed this namespace because, the autoconfig code will reside in azure-spring-boot repo. I will add new namespace once the code is ported there and merged (possibly for next release)